src/libs/ReportUtils.js

```diff
@@ -638,7 +638,7 @@ function findLastAccessedReport(reports, ignoreDomainRooms, policies, isFirstTim
  * @returns {Boolean}
  */
 function isArchivedRoom(report) {
-    return report && report.statusNum === CONST.REPORT.STATUS.CLOSED && report.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED;
+    return report && report.statusNum === CONST.REPORT.STATUS.CLOSED && report.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report.type !== CONST.REPORT.TYPE.EXPENSE;
```

Comment:
Ok, haha, it's making somewhat more sense now. We really repurposed the "CLOSE" action for chats and now we're in a weird spot where we need to check to see if this is a chat...

Small thing - but let's check if `report.type == 'chat'` instead of "not expense report"?